website/tools/build/pkg/hugo/hugo.go

```diff
@@ -103,8 +102,7 @@ func createFileFromTemplate(templatePath string, output string, tp *TemplateData
 
 // CreateContentSingleDevice generates the hugo content files in markdown
 func CreateContentSingleDevice(dir config.Dir, drs devicerepository.Store) error {
-
-	_ = os.Mkdir(dir.Hugo.Devices, 0700)
+	_ = os.Mkdir(dir.Hugo.Devices, 0o700)
```

Comment:
Out of interest @adriansmares what is the need for the lowercase `o` ?

Reply:
It doesn't change anything semantically, it just makes things consistent with the rest of the notations that exist in Go (`0x` for hex, `0b` for binary). The formatter (`gofumpt`) does this automatically for consistency purposes. 

I recommend setting it in your VSCode settings as the formatter for Go in order to save time.